orca-core/src/main/java/com/netflix/spinnaker/orca/StageResolver.java

```diff
@@ -48,6 +53,13 @@ public StageResolver(Collection<StageDefinitionBuilder> stageDefinitionBuilders)
         stageDefinitionBuilderByAlias.put(alias, stageDefinitionBuilder);
       }
     }
+
+    if (!Objects.equals(apiStages, null)) {
```

Comment:
If you want to test if `apiStages` is null, just use `apiStages != null` or just filter out in the loop. 
If you want to test if any stage is null, use something like:
```
if (apiStages.stream().anyMatch(s -> s == null)) {
...
}
```

You could also just:
```
apiStages.stream().forEach(s -> stageDefinitionBuilderByAlias.put(s.getName(), new SimpleStageDefinitionBuilder(s));
```